conjur/logic/credential_provider/keystore_credentials_provider.py

```diff
@@ -80,6 +81,16 @@ def remove_credentials(self, conjurrc):
         """
         Method for removing user credentials in the system's keyring
         """
-        logging.debug(f"Attempting to remove credentials from the '{KeystoreAdapter.get_keyring_name()}' credential store...")
-        KeystoreAdapter.delete_password(conjurrc.conjur_url)
-        logging.debug(f"Successfully removed credentials from the '{KeystoreAdapter.get_keyring_name()}' credential store...")
+        logging.debug("Attempting to remove credentials from "
+                      f"the '{KeystoreAdapter.get_keyring_name()}' credential store...")
```

Comment:
```suggestion
                      f"the '{KeystoreAdapter.get_keyring_name()}' credential store")
```

still needs to be removed